tests/unit/Type/Exception/TypeAssertExceptionTest.php

```diff
@@ -47,11 +40,6 @@ public function testIncorrectResourceType(): void
             static::assertSame('resource (curl)', $e->getExpectedType());
             static::assertSame('resource (stream)', $e->getActualType());
             static::assertSame('Expected "resource (curl)", got "resource (stream)".', $e->getMessage());
-
-            $trace  = $e->getTypeTrace();
-            $frames = $trace->getFrames();
-
-            static::assertCount(0, $frames);
```

Comment:
I'm not sure if I'm seeing this correctly, but is there a test verifying a composite exception on a subpath somewhere? Would be good to keep the error message somehow descriptive / visible in the test suite.

Could be that it already exists, but isn't part of this diff :thinking:

Reply:
That's on my radar. I'm trying to get it fast and thrustworthy first and will cover missing test cases later in this PR.